PROCESSORS.md

```diff
@@ -433,10 +436,15 @@ In the list below, the names of required properties appear in bold. Any other pr
 
 ### Output Attributes
 
-| Attribute   | Relationship | Description               |
-|-------------|--------------|---------------------------|
-| mqtt.broker |              | URI of the sending broker |
-| mqtt.topic  |              | Topic of the message      |
+| Attribute            | Relationship | Description                                                                                                                           |
+|----------------------|--------------|---------------------------------------------------------------------------------------------------------------------------------------|
+| mqtt.broker          |              | URI of the sending broker                                                                                                             |
+| mqtt.topic           |              | Topic of the message                                                                                                                  |
+| mqtt.topic.segment.n |              | The nth topic segment of the message                                                                                                  |
+| mqtt.qos             |              | The quality of service for this message.                                                                                              |
+| mqtt.isDuplicate     |              | Whether or not this message might be a duplicate of one which has already been received.                                              |
+| mqtt.isRetained      |              | Whether or not this message was from a current publisher, or was "retained" by the server as the last message published on the topic. |
```

Comment:
I'm only seeing mqtt.broker in the flow file attributes with the record reader/writer set, but I'm seeing similar record attributes with an underscore prefix. We should make it clear which attribute ends up where.